NeuroID/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.neuroid.tracker">
 
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
If the app has permission does the sdk also need this? I'm able to get the location even if this permission is commented out.

Reply:
If the user sets this permission in their app manifest, then we get it even id this is removed. Maybe we can remove this one and ask the user to add the in docs. I'll remove it here and make a ticket to update the docs.